run/nara/copy_nara.py

```
@@ -0,0 +1,237 @@
+    print(f"Instance:  {INSTANCE_ID}")
+    print(f"Source:    {src}")
+    print(f"Dest:      {dst}")
+    print(f"Temp dir:  {tmp_dir} (deleted after upload)")
+    print()
+
+    # Check if destination already has files
+    step(1, f"Verify source files in {src}")
+    existing = aws(["s3", "ls", dst]).strip() if True else ""
+    try:
+        existing = aws(["s3", "ls", dst])
+        existing_count = len([l for l in existing.splitlines() if l.strip()])
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)